test_runner/src/main/kotlin/ftl/args/AndroidArgs.kt

```diff
@@ -79,6 +81,8 @@ AndroidArgs
     }
 }
 
+val AndroidArgs.isShouldAutoGrantPermissions
+    get() = grantPermissions.isNotNull() && (grantPermissions.equals("all"))
```

Comment:
How about renaming it to match gcloud's `dontAutograntPermissions` (don't need to be exactly the same name though). Also, I think we could use an enum instead of string values.

```suggestion
    get() = grantPermissions.isNotNull() && grantPermissions.equals("none")
```

Reply:
for two enums I am not convinced? why do you think so?